app/studentApi.js

```diff
@@ -0,0 +1,66 @@
+"use strict"
+
+// dependencies
+const express = require("express");
+var router = express.Router();
+var router = require('./router.js');
+const studentDB = require("./studentDB.json");
+
+// private
+function checkStudents(studentId) {
+    return studentsDB.find(function(student){student.id === studentID;});
+}
+
```

Comment:
Why you have two function with the same name in the same file?

![image](https://cloud.githubusercontent.com/assets/5324414/21114286/bd08d3ca-c093-11e6-9d2e-924f96c3830c.png)